wzdx/raw_to_standard/geotab_avl.py

```diff
@@ -0,0 +1,227 @@
+import argparse
+import copy
+import json
+import logging
+import time
+import uuid
+from collections import OrderedDict
+
+from wzdx.tools import date_tools, polygon_tools, wzdx_translator, cdot_geospatial_api
+from wzdx.util.collections import PathDict
+
+PROGRAM_NAME = 'GeotabAvlRawToStandard'
+PROGRAM_VERSION = '1.0'
+
+STRING_DIRECTION_MAP = {'north': 'northbound', 'south': 'southbound',
+                        'west': 'westbound', 'east': 'eastbound'}
+
+REVERSED_DIRECTION_MAP = {'northbound': 'southbound', 'southbound': 'northbound',
+                          'eastbound': 'westbound', 'westbound': 'eastbound'}
+
+DISTANCE_AHEAD = 2
+
+
+# Look at planned events messages
+# Find planned event that links to automated attenuator
+#
+
+
+def main():
```

Comment:
Doesn't look like we have any unit tests for this file, can we get those?

Reply:
removed this file, it was old